### ISSUE TITLE
Can't find variable $

### ISSUE THREAD

**Issue by drinchev** [fantattitude/refined-github-safari#1]
Hey there,

I was trying to use the extension ( Safari 9.1 / OS X 10.11.4 ), but it seems that it can't find jQuery.

```
ReferenceError: Can't find variable: $ 
global code — refined-github.js (line 259)
```

Do you know is this a Safari issue or GitHub itself.

**Comment by fantattitude** [comment]
Yeah I have the same, didn't see it. Maybe it's something particular in Chrome. I'll look into it, adding jQuery as a dependency if necessary.

**Comment by fantattitude** [comment]
Ok I did the mistake, jQuery is needed, it's a dependency of the plugin. I'll add it in the next releases.

**Comment by ruimarinho** [comment]
@fantattitude any chance you can look at this? Really missing this extension on Safari :)

**Comment by fantattitude** [comment]
Yeah sorry, trying to fix the problem with jQuery (I did) but can't fix a problem when babel converts a part of the file. I'll push my branch so people can try to fix it.
